app/views/layouts/application.html.slim

```diff
@@ -32,7 +32,7 @@ html.govuk-template lang='en'
     = navigation
 
     .govuk-width-container
-      = govuk_phase_banner(tag: { text: 'Beta' }, classes: 'noprint') do
+      = govuk_phase_banner(tag: { text: content_for?(:hide_back_link) ? "true" : "false" }, classes: 'noprint') do
```

Comment:
@jack-coggin @ebrett this is an area where your PRs will clash. How do you want to proceed?